tests/x11/rmt/rmt_feature.pm

```diff
@@ -0,0 +1,89 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Add rmt configuration test and basic configuration via
+#    rmt-wizard, test enable/disable products/repo, test rmt sync
+#    rmt mirror, test import SMT data to RMT
+# Maintainer: Yutao wang <yuwang@suse.com>
+
+use strict;
+use warnings;
+use testapi;
+use base 'x11test';
+use repo_tools;
+use utils;
+use x11utils 'turn_off_gnome_screensaver';
+
+sub run {
+    x11_start_program('xterm -geometry 150x35+5+5', target_match => 'xterm');
+    # Avoid blank screen since smt sync needs time
+    turn_off_gnome_screensaver;
+    become_root;
+    rmt_wizard();
+    # sync from SCC
+    rmt_sync;
+    # enable all modules of products
+    assert_script_run("rmt-cli p e SLES/15.2/x86_64 --all-modules");
+    assert_script_run("rmt-cli product list | grep sle-module-basesystem/15.2/x86_64");
+    assert_script_run("rmt-cli products enable --all-modules SLES/15");
+    assert_script_run("rmt-cli product list | grep sle-module-server-applications/15/aarch64");
+    assert_script_run("rmt-cli product list | grep sle-module-server-applications/15/ppc64le");
+    assert_script_run("rmt-cli product list | grep sle-module-server-applications/15/s390x");
+    assert_script_run("rmt-cli product list | grep sle-module-server-applications/15/x86_64");
+    assert_script_run("rmt-cli products disable 1946 1963 1967 1971 1982 1988 1992 1955 2075 1939 1950 1998 1973 1974 1976");
+    assert_script_run("rmt-cli products disable 1589 1588 1587 1576 1640 1641 1642 1595 1594 1593 1578 1598 1597 1596 1579 2076 2077 1604 1603 1602 1581 1645 1616 1646 1611 1601 1600 1599 1580 1599 1580 1728 1586 1585 1584 1575 1740 1741 1742 1743 1718 1719 1720 1721");
+
+    # enable product with product ID
+    assert_script_run("rmt-cli product enable 1798 1973 1974");
+    assert_script_run("rmt-cli product list | grep 1798");
+    assert_script_run("rmt-cli product list | grep 1973");
+    assert_script_run("rmt-cli product list | grep 1974");
+
+    # disable product with product ID
+    assert_script_run("rmt-cli product disable 1798 1973 1974");
+
+    # enable repo with repo ID
+    assert_script_run("rmt-cli repo enable 3393 3391");
+    assert_script_run("rmt-cli repo list | grep 3393");
+    assert_script_run("rmt-cli repo list | grep 3391");
+
+    # disable repo with repo ID
+    assert_script_run("rmt-cli repo disable 3393 3391");
+
+    # mirror packages
+    rmt_enable_pro;
+    rmt_mirror_repo();
+    assert_script_run("rmt-cli product list | grep sle-module-legacy/15/x86_64");
+
+    # import smt data
+    my $datapath = "/rmtdata/";
+    my $datafile = get_var("SMT_DATA_FILE");
```

Comment:
Where is the SMT data come from? Already upload to OSD?

Reply:
I put the SMT data to one server. We can get SMT data through 'wget'. Not OSD